pkg/scheduler/preemption/preemption_test.go

```diff
@@ -2697,10 +2675,9 @@ func TestFairPreemptions(t *testing.T) {
 				t.Errorf("Issued preemptions (-want,+got):\n%s", diff)
 			}
 
-			beforeResourceNodes := resourceNodes(beforeSnapshot)
-			afterResourceNodes := resourceNodes(snapshotWorkingCopy)
-			if diff := cmp.Diff(beforeResourceNodes, afterResourceNodes, snapCmpOpts); diff != "" {
-				t.Errorf("Snapshot was modified (-initial,+end):\n%s", diff)
+			snapshotComparer := NewSnapshotComparer(cmpopts.IgnoreFields(cache.ResourceNode{}, "Usage"))
+			if diff := snapshotComparer.Diff(beforeSnapshot, snapshotWorkingCopy); diff != "" {
+				t.Errorf("Snapshot was modified by preemption (-initial,+end):\n%s", diff)
```

Comment:
```suggestion
				t.Errorf("Snapshot was modified (-initial,+end):\n%s", diff)
```

I think we can revert here.